utils.py

```diff
@@ -243,3 +251,16 @@ def parse_details_to_file_bug(details, oldest_alert):
 
     return ({'summary':summary,'desc':desc})
 
+def find_bugnum_from_body(keyrev):
+    db = create_db_connnection()
+    cursor = db.cursor()
+    query = "select id,body from alerts  where keyrevision = '%s'" % keyrev
+    cursor.execute(query)
+    search_results = cursor.fetchall()
+    bugs=set(re.findall(r'- [Bb]ug ([0-9]+)', str(search_results)))
+    logging.debug(sorted(bugs))
+    cursor.close()
+    db.close()
+
+    return sorted(bugs)
```

Comment:
sorted bugs will not get us the first bug in the list when we do bugs[0], I think we can skip the sorted() call